db/schema.rb

```diff
@@ -323,6 +323,9 @@
     t.integer  "creator_id",         limit: 4,   null: false
     t.datetime "created_at",                     null: false
     t.datetime "updated_at",                     null: false
+    t.integer  "deploy_group_id",    limit: 4,   null: false
```

Comment:
so this is a good question.  there is no real reason to put these in there as long as we don't mind parsing the key to find secrets that relate to a given set of objects.  thoughts?